tests/vector.rs

```diff
@@ -87,6 +88,15 @@ macro_rules! impl_test_rem {
     )
 }
 
+macro_rules! impl_test_iter_sum {
+    ($VectorN:ident { $($field:ident),+ }, $ty:ty, $s:expr, $v:expr) => (
+        // iter(vector).sum() ops
```

Comment:
Copypasta from the `div` macro; I originally though this might be more complicated :) .